kani-driver/src/args.rs

```diff
@@ -448,6 +446,13 @@ impl KaniArgs {
             ));
         }
 
+        if self.dry_run {
+            return Err(Error::raw(
+                ErrorKind::ValueValidation,
+                "We no longer support `--dry-run`. Use --verbose instead.",
```

Comment:
Suggest saying:
```
The `--dry-run` option has been deprecated. Use `--verbose` instead.
```

Reply:
Usually deprecated  means that it still works and it will be removed which isn't the case here.

Reply:
How about "obsolete"?

Reply:
Sure